README.md

```diff
@@ -91,18 +91,24 @@ You can start `x16emu`/`x16emu.exe` either by double-clicking it, or from the co
 
 * When starting `x16emu` without arguments, it will pick up the system ROM (`rom.bin`) from the executable's directory.
 * The system ROM filename/path can be overridden with the `-rom` command line argument.
+* `-prg` lets you specify a `.prg` file that gets loaded after start. It is fetched from the host filesystem, even if an SD card is attached!
+* `-bas` lets you specify a BASIC program in ASCII format that automatically typed in (and tokenized).
+* `-run` executes the application specified through `-prg` or `-bas` using `RUN` or `SYS`, depending on the load address.
```

Comment:
The only thing `-run` does is inject `RUN` into the keyboard buffer at the right time. A ML program would have to have the BASIC stub with the SYS call.

Reply:
I know this is mainly a rearranged document and this was already here, but I'm making notes here anyway for the things that I notice aren't quite accurate.

Reply:
This text is taken verbatim from x16emu -h. I'm happy to change it, but we'll need to change the help menu, as well.

Reply:
I have altered the entry in main.c and the readme to remove reference to the SYS command.